main.go

```diff
@@ -84,6 +88,8 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.DurationVar(&syncPeriod, "sync-period", 3*time.Hour, "The minimum interval at which watched resources are reconciled (e.g. 12h)")
```

Comment:
Should sync-period be added to manager.yaml so that the value becomes configurable?

Reply:
added in env variable and command line